networkx/drawing/nx_pylab.py

```diff
@@ -597,8 +599,11 @@ def draw_networkx_edges(
 
     Returns
     -------
-    list of matplotlib.patches.FancyArrowPatch
-        `FancyArrowPatch` instances of the directed edges
+    list of matplotlib.patches.FancyArrowPatch or matplotlib.colections.LineCollection
+        If ``arrows=True``, a list of FancyArrowPatches is returned.
+        If 1`arrows=False``, a LineCollection is returned.
+        If ``arrows=None`` (the default), then a LineCollection is returned if
+        `G` is undirected, otherwise returns a list of FancyArrowPatches.
```

Comment:
```suggestion
     matplotlib.colections.LineCollection or a list of matplotlib.patches.FancyArrowPatch
        If ``arrows=True``, a list of FancyArrowPatches is returned.
        If ``arrows=False``, a LineCollection is returned.
        If ``arrows=None`` (the default), then a LineCollection is returned if
        `G` is undirected, otherwise returns a list of FancyArrowPatches.
```
1 typo and maybe switch order of first line so it is clearly not a list of `LineCollection`s.